ivy/functional/frontends/jax/numpy/fft.py

```diff
@@ -70,6 +70,33 @@ def ifft2(a, s=None, axes=(-2, -1), norm=None):
     return ivy.array(ivy.ifft2(a, s=s, dim=axes, norm=norm), dtype=ivy.dtype(a))
 
 
+@to_ivy_arrays_and_back
+def ifftshift(x, axes=None):
+    # Check if an array
+    if not ivy.is_array(x):
+        raise ValueError("Input 'x' must be an  array")
+
+    # Get the shape of x
+    shape = ivy.shape(x)
+
+    # If axes is None, shift all axes
+    if axes is None:
+        axes = list(range(ivy.ndims(x)))
```

Comment:
The tests are failing because ivy doesn't have `ndims` attribute.

Reply:
I guess I missed that while reading the docs , however , I have changed it to len(shape)